nethserver-mail-common.spec

```diff
@@ -1,6 +1,6 @@
 Summary: Common configuration for mail packages
 Name: nethserver-mail-common
-Version: 1.6.6
+Version: 1.6.7
```

Comment:
This can be incremented only for a tagged release, after QA phase.

Reply:
yep my human fault, I will remove it